Energy Saver/Pages/Index.cshtml

```diff
@@ -18,23 +18,43 @@
                 <th style="text-align:right">Electricity amount</th>
                 <th style="text-align:right">Water amount</th>
                 <th style="text-align:right">Heating amount</th>
+                <th> </th>
             </tr>
         </thead>
+        @{
+            int yearIndex = 0;
+        }
         @foreach (var year in Model.Taxes)
         {
             <tr>
-                <td colspan=5 style="text-align:left"><b>@year[0].Year</b></td>
+                <td colspan=6 style="text-align:left"><b>@year[0].Year</b></td>
             </tr>
+
+            int index = 0;
             @foreach (var taxRecord in year)
             {
                 <tr>
                     <td style="text-align:left">@taxRecord.Month.ToString()</td>
-                    <td style="text-align:right">@(taxRecord.GasAmount % 1 == 0 ? (int)taxRecord.GasAmount : taxRecord.GasAmount)</td>
-                    <td style="text-align:right">@(taxRecord.ElectricityAmount % 1 == 0 ? (int)taxRecord.ElectricityAmount : taxRecord.ElectricityAmount)</td>
-                    <td style="text-align:right">@(taxRecord.WaterAmount % 1 == 0 ? (int)taxRecord.WaterAmount : taxRecord.WaterAmount)</td>
-                    <td style="text-align:right">@(taxRecord.HeatingAmount % 1 == 0 ? (int)taxRecord.HeatingAmount : taxRecord.HeatingAmount)</td>
+                    <td style="text-align:right">@(
+                taxRecord.GasAmount % 1 == 0 ? (int)taxRecord.GasAmount : taxRecord.GasAmount)</td>
```

Comment:
Nice. I thought of adding a single delete button at the bottom but this solution works well too